tests/src/providers/CMakeLists.txt

```diff
@@ -39,6 +40,13 @@ add_qgis_test(testqgspostgresexpressioncompiler.cpp MODULE provider LINKEDLIBRAR
 add_qgis_test(testqgspostgresprovider.cpp MODULE provider LINKEDLIBRARIES provider_postgres_a qgis_core)
 add_qgis_test(testqgspostgresconn.cpp MODULE provider LINKEDLIBRARIES provider_postgres_a qgis_core LABELS "POSTGRES")
 
+set(ENABLE_MSSQLTEST_CPP FALSE CACHE BOOL "Enable MsSQL provider C++ tests")
```

Comment:
first time I see it written with lowercase "s" ... :)

Reply:
That's coming from here https://github.com/qgis/QGIS/blob/master/tests/src/python/CMakeLists.txt#L1
:-)

Reply:
hehe